src/parser.js

```diff
@@ -14,10 +14,6 @@ const parser = yargs
       default: getPathInHostNodeModules('eslint'),
       describe: 'The path to the eslint module to use',
     },
-    prettierPath: {
```

Comment:
Oh, and don't remove the whole option. Just the default property. And you'll need to update snapshots.